tests/unit/dataqualityfood.t

```diff
@@ -1885,4 +1885,29 @@ ok(
 	'Soluble fiber + Insoluble fiber exceeds total fiber'
 ) or diag explain $product_ref;
 
+# Test case 1: Mozzarella category with sufficient ingredients
+my $product_ref_1 = {
+	category_id2 => 'mozzarella',
```

Comment:
this field does not exists !

Its categories_tags that you have to use (and it's a list). You must use the taxonomy id for this category (seek for a mozarella on open food facts and look at its json).